### PR TITLE
Update IDPH-Facility ETL

### PR DESCRIPTION
**Pull request by nss10**
Jira Ticket: [COV-783](https://occ-data.atlassian.net/browse/COV-783)

### New Features
- We can now get IDPH-facility data over a date range from the source.

### Improvements
- IDPH-facility source url which was deprecated since 11/06/2020 is updated with a new url

### CODE REVIEW

File: covid19-etl/etl/idph_facility.py [hunk 1]
```diff
@@ -43,61 +43,34 @@ def files_to_submissions(self):
...
-                "%Y-%m-%d"
-            ):
+            if len(data) == 0:
                 print(
-                    "Nothing to submit: latest submitted date and date from data are the same."
+                    "Nothing to submit: No new data available in the given date range."
                 )
                 return
 
-            if "LTC_Reported_Cases" in data:
```
Comment (paulineribeyre) [inline covid19-etl/etl/idph_facility.py:74]: any idea what this was?

File: covid19-etl/etl/idph_facility.py [hunk 2]
```diff
@@ -159,7 +132,7 @@ def parse_facility(self, date, facility):
             "deaths": deaths,
             "submitter_id": summary_clinical_submitter_id,
             "lastUpdateEt": date,
-            "date": date,
+            "date": etlJobDate,
```
Comment (paulineribeyre) [inline covid19-etl/etl/idph_facility.py:135]: i might have misunderstood the conversation today but i would do the opposite;
`lastUpdateEt = today (the last date this data was updated)`
and `date = facility["ReportDate"] (the date the data is for)`

File: covid19-etl/etl/idph_facility.py (hunk 1 above)
Comment (nss10) [reply covid19-etl/etl/idph_facility.py:74]: There was this key in the [previous source](https://dph.illinois.gov/sitefiles/COVIDLTC.json) which probably gave statewide cases and deaths information, which is no longer available in the [new source](https://idph.illinois.gov/DPHPublicInformation/api/COVIDExport/GetLTCFacilityHistorical)

File: covid19-etl/etl/idph_facility.py (hunk 2 above)
Comment (nss10) [reply covid19-etl/etl/idph_facility.py:135]: so, the `lastUpdateEt` must be the `etlJobDate`?

Comment (paulineribeyre) [reply covid19-etl/etl/idph_facility.py:135]: see Sean's message on slack:
"If you have an entry that is on 2020-11-13, then `date` is `2020-11-13`, but the `lastUpdatedEt` is `2021-08-23`."

**Review by paulineribeyre — APPROVED** [review]

**Pull request merged**